src/integrationTest/java/org/opensearch/test/framework/audit/AuditMessagePredicate.java

```diff
@@ -82,6 +85,26 @@ public static AuditMessagePredicate missingPrivilege(User user, String requestTy
         return auditPredicate(MISSING_PRIVILEGES).withLayer(Origin.TRANSPORT).withEffectiveUser(user).withTransportRequestType(requestType);
     }
 
+    public static AuditMessagePredicate privilegePredicateTransportLayer(
+        AuditCategory category,
+        User user,
+        String requestType,
+        String privilege
+    ) {
+        return auditPredicate(category).withLayer(Origin.TRANSPORT)
+            .withEffectiveUser(user)
+            .withPrivilege(privilege)
```

Comment:
By reading through the previous comment, I think the privilege is the actual permissions for the user who sends out the request is that correct?